content.md

```diff
@@ -6,6 +6,8 @@ In this lesson, we'll walk through the process of setting up a local development
 
 We'll cover the following key steps:
 
+0. **(Windows 10 and up) OS install**: If you want to use a Windows operating system then currently the easist option is to install an Linux distro into your alongside your Windows OS.
```

Comment:
Just say " If you want to use a Windows operating system then currently the easist option is to use WSL (Windows Subsystem for Linux"